packages/fiori/src/NotificationListItem.ts

```diff
@@ -40,6 +40,12 @@ import NotificationListItemTemplate from "./generated/templates/NotificationList
 // Styles
 import NotificationListItemCss from "./generated/themes/NotificationListItem.css.js";
 
+type NotificationListItemPressEventDetail = {
+	item: NotificationListItem;
+};
+
+type Footnote = Record<string, any>
```

Comment:
(line 48) add ";" to the end, although not a problem for lint, this is how we write types so far.